.github/workflows/ci.yml

```diff
@@ -0,0 +1,21 @@
+name: CI Workflow
+on:
+  - push
+
+jobs:
+  build:
+    name: Unit Tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          submodules: true
```

Comment:
Zsh doesn't exist on Ubuntu. It should be installed.
```yaml
     - name: Install Zsh
        if: runner.os == 'Linux'
        run: |
            sudo apt-get update
            sudo apt-get install -y zsh
```

Reply:
Thanks